debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresSnapshotChangeEventSource.java

```diff
@@ -210,7 +210,21 @@ protected Object getColumnValue(ResultSet rs, int columnIndex, Column column) th
             switch (type.getOid()) {
                 case PgOid.MONEY:
                     // TODO author=Horia Chiorean date=14/11/2016 description=workaround for https://github.com/pgjdbc/pgjdbc/issues/100
-                    return new PGmoney(rs.getString(columnIndex)).val;
+                    final String sMoney = rs.getString(columnIndex);
+                    if (sMoney == null) {
+                        return sMoney;
+                    }
+                    try {
+                        if (sMoney.startsWith("-")) {
+                            // PGmoney expects negative values to be provided in the format of "($XXXXX.YY)"
+                            final String negativeMoney = "(" + sMoney.substring(1) + ")";
+                            return new PGmoney(negativeMoney).val;
+                        }
+                        return new PGmoney(sMoney).val;
+                    }
+                    catch (SQLException e) {
+                        return sMoney;
+                    }
```

Comment:
Interesting, shouldn't the exception rather be raised?

Reply:
Good catch, I believe it should as well.